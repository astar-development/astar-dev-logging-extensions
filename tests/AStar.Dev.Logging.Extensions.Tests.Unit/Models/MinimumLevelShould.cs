using AStar.Dev.Logging.Extensions.Models;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Logging.Extensions.Models;

[TestSubject(typeof(MinimumLevel))]
public class MinimumLevelShould
{

    [Fact]
    public void ContainTheExpectedProperties()
    => new MinimumLevel().ToJson().ShouldMatchApproved();
}