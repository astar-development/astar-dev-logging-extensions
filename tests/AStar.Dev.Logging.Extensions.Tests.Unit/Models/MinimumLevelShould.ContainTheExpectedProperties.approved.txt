{
  "default": "Verbose",
  "override": {
    "microsoftAspNetCore": "",
    "systemNetHttp": "",
    "aStar": ""
  }
}